Tasks/CopyFilesOverSSHV0/utils.ts

```diff
@@ -28,12 +28,15 @@ export function pathIsUNC(path: string): boolean {
  * @returns {string} OS specific command to clean target folder on the remote machine
  * @param {string} targetFolder path to target folder
  * @param {boolean} forWindows return command for Windows CMD
+ * @param {boolean} cleanHiddenFiles clean hedden files in target folder
  */
-export function getCleanTargetFolderCmd(targetFolder: string, forWindows: boolean): string {
+export function getCleanTargetFolderCmd(targetFolder: string, forWindows: boolean, cleanHiddenFiles: boolean = false ): string {
     if (forWindows) {
+        const cleanFilesInTarget = cleanHiddenFiles ? `del /q /A:H "${targetFolder}\\*"` : `del /q "${targetFolder}\\*"`
```

Comment:
Nit: could it make sense to interpolate '/A:H' only?

Reply:
Yep, updated